src/packages/tiptap/extensions/toolbar/link.extension.ts

```diff
@@ -75,7 +75,7 @@ export default class UmbTiptapToolbarLinkExtensionApi extends UmbTiptapToolbarEl
 
 		const anchor = this.#getAnchorFromQueryString(queryString);
 
-		if (anchor) url += anchor;
+		if (anchor) url = (url ?? '') + anchor;
```

Comment:
In general I would not recommend writing if-sentences without the { }, I think it can be acceptable if its a return statement like this:

if (!missingThing) return;

But all other cases offen ends up with someone misunderstanding the scope of that code-block.
And then I also find it easier to read with { }

:-D